### DISCUSSION TITLE
Code completion not working until a reload

### DISCUSSION THREAD

**Discussion by NoahELE** [redhat-developer/vscode-java#3798] [General]
From time to time, the code auto completion stop suggest java standard library types like `String`, `Math`, `System`, etc. when I open a java project. Usually a reload of vscode will solve the problem. However, the suggestion for dependencies are shown correctly.

Anyone experiencing the same problem? I cannot trigger the bug consistently, thus I am not sure if I should create an issue.

**Comment by snjeza** [comment]
Related tickets::
- https://github.com/redhat-developer/vscode-java/issues/3715
- https://github.com/redhat-developer/vscode-java/issues/3793
- https://github.com/redhat-developer/vscode-java/issues/3797
